### ISSUE TITLE
False positive for min and max

### ISSUE THREAD

**Issue by takenori-y** [cpplint/cpplint#296]
Version: 2.0.0

The following code
```cc
// Copyright 2024 test
int main() {
  int max(0);
  return 0;
}
```
causes the following error.
```
test.cc:3:  Add #include <algorithm> for max  [build/include_what_you_use] [4]
Done processing test.cc
Total errors found: 1
```
`min` has the same issue.
